src/components/application_manager/test/policy_handler_test.cc

```diff
@@ -3031,10 +3032,33 @@ TEST_F(PolicyHandlerTest, GetAppPropertiesStatus_NicknameChanged_SUCCESS) {
             policy_handler_.GetAppPropertiesStatus(properties, kPolicyAppId_));
 }
 
-TEST_F(PolicyHandlerTest, GetAppPropertiesStatus_NicknameNotChanged_SUCCESS) {
+TEST_F(PolicyHandlerTest,
+       GetAppPropertiesStatus_NicknameRemoveChanged_SUCCESS) {
```

Comment:
@SNiukalov May be ..._RemoveChangedNickname_... ?

Reply:
@AByzhynar 
Thanks, fixed in:https://github.com/smartdevicelink/sdl_core/pull/3336/commits/e8b6796277ec4c559a26fb8e02562d0929e67b94